modules/virt-cnv-network-alerts-networkaddonsconfignotready.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts-networkaddonsconfignotready_{context}"]
+= NetworkAddonsConfigNotReady alert
+
+The cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster. This alert occurs when the CNAO CR NetworkAddonsConfig is not ready, indicating one of the deployed components is not ready.
+
+If CNAO components are not deployed and ready, some kubevirt networking scenarios cannot be executed.
+
+.Procedure
+
+* Diagnose the failed component deployment's cause.
+
+** Check for messages on the `networkaddonsconfig` CR status.
++
+[source,terminal]
+----
+kubectl get networkaddonsconfig -o custom-columns="":.status.conditions[*].message
+----
+
+** Identify which component is not ready and gather more information from it.
+*** Extract the component deployment or daemonSet and namespace from the message.
++
+For example, if the message is `DaemonSet "cluster-network-addons/macvtap-cni" update is being processed...`, the namespace is `cluster-network-addons` and DaemonSet name is `macvtap-cni`.
```

Comment:
macvtap-cni is not deployed on D/S. Could we use cluster-network-addons/kube-cni-linux-bridge-plugin instead?